src/main/webapp/view/discussion/discussion.js

```diff
@@ -36,7 +36,11 @@ const SELECTOR_CANCEL_REPLY = '#cancel-reply';
 const SELECTOR_POST_REPLY = '#post-reply';
 const SELECTOR_REPLY_TEXTAREA = '#reply-textarea';
 
+// 10 seconds.
+const TIME_TOLERANCE_MILLISECONDS = 10 * 1000;
+
 let newCommentTimestampMilliseconds = 0;
+let currentRootDiscussionComments = [];
```

Comment:
Can you add an inline comment for the array type? 
https://google.github.io/styleguide/jsguide.html#features-declare-types-as-needed

Reply:
Added!